### ISSUE TITLE
Invalid tsconfig.json

### ISSUE THREAD

**Issue by saeedseyfi** [fluffynuts/synchronous-promise#38]
Hi!
The tsconfig.json file doesn't contain a valid json. The issue should be related to `//` comment syntax.
Error:
```
Path: 
 ./node_modules/synchronous-promise/tsconfig.json
Message:
 Failed to parse JSON file
```

**Comment by saeedseyfi** [comment]
Uh just remembered JSON doesn't support any comment syntax. Not an issue related to synchronous-promise then. Closing the issue.

**Comment by fluffynuts** [comment]
no worries - perhaps the tsconfig.json shouldn't be in the packaged files? I don't think it provides any value?

**Comment by saeedseyfi** [comment]
Yes that can be an option. Since tsconfig is only used during development can be ignored in packaged files.

**Comment by fluffynuts** [comment]
Should be resolved with 2.0.16
